azure-iot-device/tests/provisioning/aio/test_async_sk_provisioning_device_client.py

```diff
@@ -0,0 +1,141 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
I have some quibbles about some of the test structure and naming here, but we can revisit them after - once iothub is consistent, we can bring dps up to par.

Reply:
okay

---
In reply to: [289561458](https://github.com/Azure/azure-iot-sdk-python-preview/pull/98#discussion_r289561458) [](ancestors = 289561458)